tools/csv-gen/csv-gen.go

```diff
@@ -0,0 +1,690 @@
+	templateStruct.SetAnnotations(
+		map[string]string{
+			"createdAt":           time.Now().Format("2006-01-02 15:04:05"),
```

Comment:
seems like an arbitrary date, do we have/need to use a date from 14 years ago?

Reply:
This sounds very strange, but the go follow this format this way and while time.Now() will give the current date, with that format.

Reply:
yeah that was just my failure to look up the go doc, sorry. I found the relevant section:

> Layouts must use the reference time Mon Jan 2 15:04:05 MST 2006 to show the pattern with which to format/parse a given time/string

Just an unusual break way from the typical placeholder parsing style (mm DD YY hh HH, etc), so it caught me offguard.